vars/validateArtifacts.groovy

```diff
@@ -9,7 +9,21 @@
 
 /**
 Wrapper that runs validation.sh script with provided args.
-@param Map[<any>] <Required> - Any arguments that you want to be passed to validation.sh script. eg: version: 1.0.0 will be passed as --version 1.0.0
+[
+@param Map[version] <Required> - Version of the opensearch artifact that needs to be validated.
+@param Map[file_path] <Required> - Url paths to Opensearch or OpenSearch DashBoards artifacts.
+Note: These parameters are mutually exclusive. Provide either of 'version' or  'file_path' parameters, to proceed with the validation.
+]
```

Comment:
I dont think the comment here take effect?
Or is it just the github gui?

Reply:
No it's due to GUI, All the info is inside the comments. We should be good.